docs/user.md

```diff
@@ -646,7 +646,13 @@ spec:
 
 ## In-place major version upgrade
 
-Starting with Spilo 13, operator supports in-place major version upgrade to a higher major version (e.g. from PG 10 to PG 12). To trigger the upgrade, simply increase the version in the manifest. It is your responsibility to test your applications against the new version before the upgrade; downgrading is not supported. The easiest way to do so is to try the upgrade on the cloned cluster first. For details of how Spilo does the upgrade [see here](https://github.com/zalando/spilo/pull/488), operator implementation is described [in the admin docs](administrator.md#minor-and-major-version-upgrade).
+Starting with Spilo 13, operator supports in-place major version upgrade to a
```

Comment:
The upgrade capability also depends on the operator version, don;t forget to mention it here.

Reply:
I actually think we dont need to specify either, this advertises to new users where we should assume starting with current release.